tui/src/scala/tui/widgets/ListWidget.scala

```diff
@@ -72,24 +71,16 @@ case class ListWidget(
 
   type State = ListWidget.State
 
-  def render(area: Rect, buf: Buffer, state: State): Unit = {
+  def render(area: Rect, buf: Buffer): Unit = {
```

Comment:
Alright so I think the issue is in here. Inside of this render we're not taking into account if `highlightStyle` or not is set. So when the user has a selected item, you can no longer see the desired applied style.

Reply:
See this is what puzzles me about this codebase as well. The highlightStyle is not applied where you would expect it to, but after the list element has been rendered.
This is what I was getting at with #26, heh

Reply:
so at line 134. unless you meant something else?

Reply:
Ahhh dang, I totally missed that on that line! Huh, weird then something else funky is going on that I need to figure out why it's not showing for me.

Reply:
Actually this one was my bad, I forget to pass the state directly into the `ListWidget`.